tpe/plugin/src/EntityManagement_TEST.cc

```diff
@@ -79,6 +79,17 @@ TEST(EntityManagement_TEST, ConstructEmptyWorld)
   EXPECT_EQ(model, link->GetModel());
   EXPECT_EQ(link, model->GetLink(0));
   EXPECT_EQ(link, model->GetLink("empty link"));
+
+  EXPECT_EQ(0u, model->GetNestedModelCount());
+  auto nestedModel = model->ConstructEmptyNestedModel("empty nested model");
+  EXPECT_EQ(1u, model->GetNestedModelCount());
+  ASSERT_NE(nullptr, nestedModel);
+  EXPECT_EQ("empty nested model", nestedModel->GetName());
```

Comment:
```suggestion
  EXPECT_EQ("empty nested model", nestedModel->GetName());
  EXPECT_EQ(nestedModel, model->GetNestedModel("empty nested model"));
  EXPECT_EQ(nestedModel, model->GetNestedModel(0));
```
Can we add in some tests for the two new `GetNestedModel(` methods? It looks like you've already added tests for `GetNestedModelCount` and `ConstructEmptyNestedModel`, so I believe the two `GetNestedModel` methods are the only ones missing.

Reply:
This actually failed and I realized I had fixed it in the next PR (#230), so I brought in the relevant patch in https://github.com/ignitionrobotics/ign-physics/pull/229/commits/e5679df5d6698b29f31fc34d37428cbceb8cfc52. Do you mind taking one more look?